fairing/request_test.go

```diff
@@ -0,0 +1,44 @@
+package fairing_test
+
+import (
+	"net/http"
+	"testing"
+
+	asserter "github.com/dannypsnl/assert"
+
+	"github.com/dannypsnl/rocket/fairing"
+)
+
+func TestFairingRequestCanModifyRequest(t *testing.T) {
+	assert := asserter.NewTester(t)
+	hook := fairing.OnRequest(func(r *http.Request) *http.Request {
+		assert.Eq(r.Method, "GET")
```

Comment:
I don’t think test the value what you set is needing, it would make you have to change the test when update the value you set

Reply:
I want to make sure the request is same as we expected before modify.

Reply:
I know but basically you set it up by external code, we shouldn’t test external lib, and it make test hard to read